README.md

```diff
@@ -191,4 +191,13 @@ ARM_CLIENT_ID, ARM_TENANT_ID,ARM_SUBSCRIPTION_ID and ARM_CLIENT_SECRET
 Push you changes and wait for travis to finish
 Load the azure application URL in the browser
 
+## Addition of loggy log management
+The application requires the loggy token to be created as environment variables -
+The Log level variable can be set to INFO,DEBUG,ERROR
+```
+[LOGGLY_TOKEN]
```

Comment:
In reality you would have definitely wanted to add a LOGGLY_TOKEN to your .tf file/pipeline so the "production" app has logging